### ISSUE TITLE
Make it clear when a draft post is an edit or a new post

### ISSUE THREAD

**Issue by nolanw** [Awful/Awful.app#282]
### Steps to reproduce

1. Go into any open thread where you've posted.
2. Tap the … button on one of your posts.
3. Tap "Edit".
4. Delete the contents of the new post.
5. Tap "Cancel".
6. Tap the "Compose" button on the top right of the thread.

### Expected result

I'm writing a new post? Or I'm informed that my edited post is still a thing? Or I get to choose what to do?

### Observed result

I'm still editing that old post, which is completely nonobvious and probably causes me to simultaneously delete the old post content and not make the new post I wanted.

### Quotes from the thread

> If you "edit" a post then change your mind and decide to reply and make a new one it still only edits the original post.

> I went to a post of mine intending to quote it but I hit Edit instead. When I backed out and then went to Quote, the quote got tacked on to the original post as an edit, it didn't make a new post.

> Pretty sure this is a known bug but, uh, it can be a bit more dangerous when it happens to moderators. I accidentally a guy's post today.
> — [Fedule](http://forums.somethingawful.com/showthread.php?threadid=3510131&userid=0&perpage=40&pagenumber=224#post461295184)

> if you edit a post, then hit cancel, then go quote a post, you will actually end up editing the post you were editing earlier. (rather than making a new post with quote as intended )

> So this is really minor but it would be nice if the "Cancel" button to the top and left of me right now didn't say Cancel but instead said something like "Back" since its not really a cancel button. For the longest time I didn't know I could hit that button mid post in order to grab another quote or re-read a post before posting cause I always thought it would cancel and delete my entry.

> Just encountered a fun bug...I accidentally tapped Edit on one of my own posts, so I backed out with Cancel and tapped Quote instead. Was presented with my post in Quote markup...but when I posted..it edited the original post instead of making a new one.

> Just had a weird issue. I posted a reply to a thread, went back and edited that reply. Then I went to edit it again, but cancelled the edit. Someone then replied to my post so I went to quote him (and I'm sure I clicked on the right post). Instead of seeing his post quoted in the input box I saw the text of the post I'd edited a few minutes before. I selected it and deleted it, the wrote in my response to him. I submitted it and then discovered that it just edited my previous post instead of posting a new post.
> 
> TL;DR: I guess in this case quote=edit.
> 
> Oh yeah, I'm on the latest beta of awful on whatever the latest version of iOS 10 is on an iPhone 7 plus.
> — [GutBomb](https://forums.somethingawful.com/showthread.php?noseen=0&threadid=3510131&perpage=40&pagenumber=256#post475816168)

> Also experiencing this, running awful beta and ios 11 though.
> — [Grassy Knowles](https://forums.somethingawful.com/showthread.php?threadid=3510131&userid=0&perpage=40&pagenumber=257#post475816356)

> An idea that we had pop up ages ago that I liked was replies/edits acting like composing mail, where you could swipe it down to dock it to the bottom of the screen, and then allow the user to add quotes that way. Destroying their input if they dismiss that pane altogether. 
> 
> Also titling can be better by appending "Replying:" or "Editing post in:" depending on which action you're taking.
> — [The Dave](https://forums.somethingawful.com/showthread.php?threadid=3510131&userid=0&perpage=40&pagenumber=257#post475825827)

**Comment by nolanw** [comment]
The Dave's idea is a super quick fix and we should do that pronto: put something in the title of the screen to indicate editing vs. replying. A nicer fix can come later.
